developers/weaviate/starter-guides/custom-vectors.mdx

```diff
@@ -8,187 +8,207 @@ image: og/docs/tutorials.jpg
-         startMarker="# START create schema"
-         endMarker="# END create schema"
-         language="py"
-      />
-   </TabItem>
-   <TabItem value="ts3" label="JS/TS Client v3">
-      <FilteredTextBlock
-         text={Ts3Code}
-         startMarker="// START create schema"
-         endMarker="// END create schema"
-         language="ts"
-      />
-   </TabItem>
-   <TabItem value="ts2" label="JS/TS Client v2">
-      <FilteredTextBlock
-         text={Ts3Code}
-         startMarker="# START create schema"
-         endMarker="# END create schema"
-         language="ts"
-      />
-   </TabItem>
-</Tabs>
+Weaviate uses the vectors you provide at import even if the collection specifies a vectorizer.
 
-## Import the data
+## Import data and vectors
 
-The example data is based on a [set of ten questions](https://raw.githubusercontent.com/weaviate-tutorials/quickstart/main/data/jeopardy_tiny_with_vectors_all-OpenAI-ada-002.json) from the "Jeopardy!" television program. Each of the data objects contains the following elements:
+The example data is based on a set of ten questions from the "Jeopardy!" television program. [The data file](https://raw.githubusercontent.com/weaviate-tutorials/quickstart/main/data/jeopardy_tiny_with_vectors_all-OpenAI-ada-002.json) is a JSON formatted document that contains the vectors and the underlying data.
 
-- A vector embedding
-- A question
-- A category
-- An answer
+For large datasets, batch import is more efficient than importing individual objects. This [batch import](/developers/weaviate/tutorials/import#batch-import-process) code imports the question objects, including their vectors.
 
-You can use any vectorizer. In this example, OpenAI is the vectorizer. The question, answer, and category are the raw data. The vector embedding is the vector OpenAI returns after processing the raw data.
+import ImportQuestionsWithVectors from '/_includes/code/quickstart.import.questions-and-vectors.mdx'
 
-Create a JSON formatted data file to use in your import. In this example, the [data import file](https://raw.githubusercontent.com/weaviate-tutorials/quickstart/main/data/jeopardy_tiny_with_vectors_all-OpenAI-ada-002.json) is already prepared. The JSON file encodes this data:
+<ImportQuestionsWithVectors />
 
-<details>
-  <summary>View the dataset</summary>
+:::tip Vectors are not object properties
+When you specify the elements of your batch import, be careful not to mislabel vectors as properties. If the vector is listed in the properties section, Weaviate processes it as a regular property, rather than as a vector embedding.
+:::
+
+## Query
+
+In order to query a vector, Weaviate needs a vector representation of the query. If the collection definition specifies a vectorizer, Weaviate uses that vectorizer to generate a vector embedding for the query. However, the example collection in this guide does not specify a vectorizer. Since Weaviate cannot vectorize the query, you have to supply a query vector.
```

Comment:
I found this a little confusing. Propose this for brevity:
> At query time, provide the query vector directly as a parameter to perform a [`nearVector`](/developers/weaviate/search/similarity#search-with-a-vector) search.

Additionally, I think it would also be good to show a hybrid search with an explicit vector input, and a RAG query.